src/main/java/uk/gov/hmcts/reform/finrem/ccddatamigration/service/CommonFunction.java

```diff
@@ -29,4 +29,10 @@ public static boolean isCaseInCorrectState(CaseDetails caseDetails, String expec
 
         return expectedState.equalsIgnoreCase(nullToEmpty(caseDetails.getState()));
     }
+
+    public static boolean isCaseInCorrectState(CaseDetails caseDetails, String expectedState1, String expectedState2) {
```

Comment:
could change to var args if we ever expect more than two states
```
    public static boolean isCaseInCorrectState(CaseDetails caseDetails, String ... states) {
     for(String state : states) {
       if(state.equalsIgnoreCase(nullToEmpty(caseDetails.getState())))
           return true
     }
    return false;
}
```
if we only ever expect two its grand though 👍